api/conf/oauth2_extensions.py

```diff
@@ -11,15 +14,23 @@ class OAuth2Authentication(BaseOAuth2Authentication):
     keyword = "Bearer"
 
     def authenticate(self, request):
-        result = super().authenticate(request)
+        user_auth_tuple = super().authenticate(request)
         if getattr(request, "oauth2_error", None):
             # oauth2_error is only defined on requests that had errors
             # it will be undefined or empty for anonymous requests and
             # requests with valid credentials
             # `request` is mutated by `super().authenticate`
             raise AuthenticationFailed()
+        # if this is an authed request, check and
```

Comment:
Very minor nit-pick for extra whitespace to help visually break the function up.

```suggestion

        # if this is an authed request, check and
```

Reply:
@sarayourfriend Updated!